postinstall/upgrade_many.py

```diff
@@ -193,6 +193,39 @@ def ssh_unchecked_call(*args):
             log("Running '%s' on %s via ssh (allow failure)" % (' '.join(str(x) for x in args), ip))
             subprocess.call(ssh_args(args))
 
+        def ssh_call_with_output(*args):
+            log("Running '%s' on %s (%s) via ssh, capturing output" % (
+            ' '.join(str(x) for x in args), hostname, ip))
+            return subprocess.check_output(ssh_args(args))
+
+        def container_has_drives():
```

Comment:
Make this function take a role instead of being hardcoded to drives.
You can then implement a wrapper for the generic function

Reply:
sure